lib/procore/auth/access_token_credentials.rb

```diff
@@ -38,7 +38,7 @@ def client
         OAuth2::Client.new(
           client_id,
           client_secret,
-          site: "#{host}/oauth/token",
+          site: "#{Procore.configuration.auth_host}/oauth/token",
```

Comment:
I assume this will be set to `login.procore.com`?

Reply:
Yep its the default

https://github.com/procore/ruby-sdk/blob/411810079f8d5043b06402b8dedceb69b9c618cb/lib/procore/defaults.rb#L9-L10